saga-core/src/main/java/io/servicecomb/saga/core/application/SagaCoordinator.java

```diff
@@ -74,23 +84,29 @@ public SagaCoordinator(
       ExecutorService executorService) {
     this.persistentStore = persistentStore;
     this.requestInterpreter = new JsonRequestInterpreter(fromJsonFormat, new GraphCycleDetectorImpl<>());
+    this.fromJsonFormat = fromJsonFormat;
     this.toJsonFormat = toJsonFormat;
     this.executorService = executorService;
   }
 
   @Segment(name = "runSagaCoordinator", category = "application", library = "kamon")
-  public void run(String requestJson) {
+  public void run(String requestJson) throws IOException {
```

Comment:
we shouldn't throw checked exception out

Reply:
Done